### ISSUE TITLE
Asterisk: New releases

### ISSUE THREAD

**Issue by jospezial** [openwrt/telephony#703]
Maintainer: @jslachta
@micmac1 @dhewg 

Be aware that some asterisk modules are dropped or deprecated with asterisk 19 and 20 branch.
old https://downloads.asterisk.org/pub/telephony/asterisk/releases/asterisk-19.0.0-summary.html only to see deprecation and drops against branch 18

The next upstream LTS branch release will be 20.0.0
https://downloads.asterisk.org/pub/telephony/asterisk/releases/asterisk-20.0.0-rc1-summary.html

**Comment by jospezial** [comment]
Build with asterisk-19.0.0 worked without changing our patches.
No regressions so far. I did  make calls to an echo test number, but not to real people now at night.
```
-PKG_VERSION:=18.7.1
+PKG_VERSION:=19.0.0
 PKG_RELEASE:=$(AUTORELEASE)
 
 PKG_SOURCE:=asterisk-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.asterisk.org/pub/telephony/asterisk/releases
-PKG_HASH:=11c8728718b59af74440276ba888484bc9201217719d56436e1e0e85bf533c17
+PKG_HASH:=8fd2871d686dd9270c6abe5f3c3809b2b3a6248e2560b2c2354f2244119ff92b

```

**Comment by jospezial** [comment]
Build with asterisk-19.1.0 worked without changing our patches.
No regressions so far on lantiq.
@micmac1 Do you like to create a PR for the new versions and check which modules to remove from our Makefile for asterisk-19 branch?

asterisk-18.9.0 for telephony 21.02 and asterisk-19.1.0 for telephony master?
```
diff --git a/net/asterisk/Makefile b/net/asterisk/Makefile
index afa5c24..c22b9d6 100644
--- a/net/asterisk/Makefile
+++ b/net/asterisk/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=asterisk
-PKG_VERSION:=18.7.1
+PKG_VERSION:=19.1.0
 PKG_RELEASE:=$(AUTORELEASE)
 
 PKG_SOURCE:=asterisk-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.asterisk.org/pub/telephony/asterisk/releases
-PKG_HASH:=11c8728718b59af74440276ba888484bc9201217719d56436e1e0e85bf533c17
+PKG_HASH:=3c2755e9c037f2b8e25e7ccee22c51e7353968bf9f845fcbf0dafef0850986dd
 
 PKG_BUILD_DEPENDS:=libxml2/host
 
```

**Comment by micmac1** [comment]
Hello Jo,

Me personally, I don't have the time to try always catching up with new releases. I make time when Asterisk upstream announces security issues. Apart from that we stick to LTS releases. The next upstream LTS release is probably going to be 20.

**Comment by Neustradamus** [comment]
@jospezial: @micmac1 has done a PR here:
- https://github.com/openwrt/telephony/pull/776

**Comment by jslachta** [comment]
This is fixed in PR #785
